docs/reference/components/sources/mongodb_metrics.cue

```diff
@@ -0,0 +1,801 @@
+package metadata
+
+components: sources: mongodb_metrics: {
+	title:             "MongoDB Metrics"
+	long_description:  "[MongoDB][urls.mongodb] is a general purpose, document-based, distributed database built for modern application developers and for the cloud era."
+	short_description: "Collect metrics from a MongoDB server."
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["daemon", "sidecar"]
+		development:   "beta"
+		egress_method: "batch"
+		function:      "collect"
+	}
+
+	features: {
+		checkpoint: enabled: false
+		multiline: enabled:  false
+		tls: enabled:        false
+	}
+
+	support: {
+		platforms: {
+			"aarch64-unknown-linux-gnu":  true
+			"aarch64-unknown-linux-musl": true
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: [
+			"User from endpoint should have enough privileges for running [serverStatus][urls.mongodb_command_server_status] command",
+		]
+
+		warnings: []
+		notices: []
+	}
+
+	configuration: {
+		endpoint: {
```

Comment:
Yeah, it would be nice to have nearly the same options. @jamtur01 should I change to `endpoint` => `endpoints` in mongodb source? Or we will go with `endpoint` and will and `endpoints` later if this will be required?

Reply:
Let's update it now for consistency.